source/_components/sensor.github.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title: "GitHub Sensor"
+description: "How to integrate the GitHub Sensor into Home Assistant."
+date: 2018-12-24 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: github.png
+ha_category: Sensor
+ha_release: 0.85
+ha_iot_class: "Cloud Polling"
+---
+
+The GitHub sensor integrates data from [GitHub](https://github.com/) to monitor your favorite repositories.
+
+## {% linkable_title Setup %}
+
+You will need your GitHub username and password. You can also create an access token [here](https://github.com/settings/tokens). An access token is required if you use two factor authentication with your account.
+
+## {% linkable_title Configuration %}
+
+To enable this platform, please add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: github
+    access_token: !secret github_access_token
```

Comment:
Please remove all non required options from the example.

Reply:
Technically, either username/password or an access_token is required. Since either/or is required, what should I state here? Examples for both methods?

Reply:
Good question... preferably the user will use `access_token` for communication, so maybe this part should be as is.